src/tcldcc.c

```diff
@@ -1176,15 +1176,22 @@ static int setlisten(Tcl_Interp *irp, char *ip, char *portp, char *type, char *m
     if (strlen(newip)) {
```

Comment:
Looking at line 1039, I actually think this branch logic can be removed, as we always set newip now? If you can double-check me on that, I'll clean this section up appropriately.